src/main/resources/db/migration/local+dev/R__6_create_rooms_and_beds.sql

```diff
@@ -0,0 +1,83 @@
+-- ${flyway:timestamp}
+
+TRUNCATE TABLE rooms CASCADE;
+TRUNCATE TABLE beds CASCADE;
+
+-- approved_premises (placeholder)
+
+-- temporary_accommodation_premises
+--      Premises 1 (2x rooms, 1x bed each)
+  insert into
+    rooms ("id", "name", "notes", "premises_id")
+  values
+    (
+      '14c0911e-6296-4b3f-ad00-5a2cf6f23a08',
+      'ROOM1',
+      NULL,
+      'd6447105-4bfe-4f1e-add7-4668e1ca28b0'
```

Comment:
I should say that these premises ids come from the TA seed file that runs before hand https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/src/main/resources/db/migration/local%2Bdev/R__2_create_premises.sql#L31